src/python/pants/backend/python/subsystems/python_tool_base.py

```diff
@@ -34,11 +36,15 @@ class PythonToolRequirementsBase(Subsystem):
     # Subclasses do not need to override.
     default_extra_requirements: ClassVar[Sequence[str]] = []
 
+    # Subclasses may set to override the value computed from default_version and
+    # default_extra_requirements.
+    # TODO: Once we get rid of those options, subclasses must set this.
+    default_package_names: Sequence[str] = []
+
     default_interpreter_constraints: ClassVar[Sequence[str]] = []
     register_interpreter_constraints: ClassVar[bool] = False
 
-    # If this tool does not mix with user requirements (e.g. Flake8 and Isort, but not Pylint and
```

Comment:
We generate lockfiles for all of those tools (in fact for all tools, period). So this comment was out of date.